components/org.wso2.carbon.editor.log.appender/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Please change the licence header's year

Reply:
Removed log4j2.xml by commit [#d896f9ef4517910b3bb073bf6b2a1c6fce2aacf3](https://github.com/wso2/carbon-analytics/pull/797/commits/d896f9ef4517910b3bb073bf6b2a1c6fce2aacf3)